test/test_ops.py

```diff
@@ -282,6 +282,9 @@ def test_round(self):
     helper_test_op([(45,35)], lambda x: x.round(), forward_only=True)
     helper_test_op(None, lambda x: x.round(), vals=[[1.499, 1.5, 1.501, 1.0, 2.1, 0.0, -5.0, -2.499, -2.5, -2.501]], forward_only=True)
     helper_test_op(None, lambda x: x.round(), vals=[[2.5, -1.5]], forward_only=True)
+  def test_lerp(self):
+    helper_test_op([(45,35), (45,35), (45,35)], lambda x,y,z: x.lerp(y,z), forward_only=True)
```

Comment:
Why forward_only?

Reply:
The rest of similar (math) function tests were forward_only, so I assumed it was because grads are checked already with 'primitives'. But I can make it also check the derivatives if you want

Reply:
Does it work?

Reply:
wdym 'work'? The function itself is just some binary ops, it works fine. The test also passes.

Reply:
The backward pass

Reply:
Ah, yeah, it works fine. It's also not problematic since it's just some binary ops together.

Reply:
Test it?

Reply:
activated backward tests for it